testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -191,9 +211,17 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
     cluster_topology = params_from_base_suite_setup["cluster_topology"]
     mode = params_from_base_suite_setup["mode"]
     xattrs_enabled = params_from_base_suite_setup["xattrs_enabled"]
+    sg_lb = params_from_base_suite_setup["sg_lb"]
 
     test_name = request.node.name
 
+    if sg_lb:
```

Comment:
Can we be more explicit with the test names that we are skipping so we don't accidentally skip tests.